lib/model.js

```diff
@@ -484,6 +484,7 @@ var validateIncludedElements = function(options, tableNames) {
         include.subQuery = false;
       } else {
         include.subQueryFilter = false;
+        include.subQuery = include.subQuery || include.hasParentRequired && (include.hasDuplicating || include.hasRequired);
```

Comment:
I suggest `hasParentRequired`?
I can't really see why it's needed, whether or not an include parent is in the subquery doesn't matter unless the specific include needs to be in the subquery (at which point the parent should be forced, but that happens at the parent level).